src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef, useContext } from 'react';
+
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+const Notification = forwardRef(() => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+  return (
+    <ThemeContext.Provider value={theme}>Notification</ThemeContext.Provider>
```

Comment:
You don't need this. I doubt we will be changing the theme value for children of Notification.